### DISCUSSION TITLE
Issue with CSP-nonce at Angular

### DISCUSSION THREAD

**Discussion by EngHema82** [primefaces/community#1160] [PrimeNG]
•	I have Updated Angular from 14 to 17 to use feature “CSP-nonce” .
•	The current Prime NG I am using is  “14.0.0” .
•	If I try running the portal , I got the following error :
[
![Capture](https://github.com/primefaces/community/assets/159539232/da8d05e6-afbb-4844-9aa5-f08a1a5978ee)
](url)

How Can I use Prime NG if I enable feature “CSP-nonce” at angular ?

**Comment by cetincakiroglu** [comment]
Hi,

I guess the image url is broken, it redirects to Github 404. Could you please update the URL so we can see the error?

**Reply by EngHema82** [reply]
Please check the image :  

![image](https://github.com/primefaces/community/assets/159539232/277f1983-827f-44df-ade3-7c4a312943f4)
